icu4j/main/core/src/test/java/com/ibm/icu/dev/test/rbbi/RemapRule.java

```diff
@@ -0,0 +1,167 @@
+// © 2024 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.dev.test.rbbi;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A segmentation rule expressed as in UAXes #14 and #29.
+ *
+ * A remap rule performs normal a regex replacement applied to the remapped
+ * string.
```

Comment:
not quite sentence is

Reply:
https://www.reddit.com/r/MEOW_IRL/comments/avqra0/meow_irl/